src/n64dd/n64dd_data_buffer.c

```diff
@@ -1,4 +1,6 @@
 #include "n64dd.h"
 
-// Buffer used for reading from the disk?
-u8 B_801DC000[0x4D10];
+// Buffer used during the process of reading from the disk
+// Used to directly read individual LBAs/blocks from the disk before copying to the final destination
+// As blocks/LBAs are read directly into this buffer from the disk, it has to be 16-byte aligned
```

Comment:
There's no need to write "block/LBA", Logical Block Addresses are more like pointers to data than the data itself, so you can just write "block" or "logical block" for this.

On alignment, 16 bytes is a recommended alignment to prevent data cache alignment but for DMA the minimum sane RAM alignment is 8 bytes, so I believe "should be" is slightly more appropriate than "has to be" in this case.

```suggestion
// Buffer used during the process of reading from the disk.
// Used to directly read individual blocks from the disk before copying to the final destination.
// As blocks are read directly into this buffer from the disk, it should be 16-byte aligned.
```